tutorial/requestcredential.js

```diff
@@ -22,9 +22,9 @@ app.use(bodyParser.json({ type: '*/*' }))
 app.get('/', function (req, res) {
 
   credentials.createRequest({
-    verified: ['Custom Attestation'],
-    callbackUrl: 'http://192.168.1.9:8081/callback',
-    exp: new Date().getTime() + 60000
+    verified: ['My Title'],
+    callbackUrl: 'http://192.168.1.14:8081/callback',
+    exp: new Date().getTime()/1000 + 300
```

Comment:
Just dividing by 1000 will result in a number that has a fractional part. Use `Math.floor()`